src/js/index.js

```diff
@@ -57,126 +27,75 @@ export default class {
     }
 
     /**
-     * Add scroll and resize listeners
+     * Set resize and optionally scroll listeners
      */
     setListeners() {
         if (this.options.loadOnScroll) {
-            window.addEventListener('scroll', helpers.throttle(() => this.setLoadOnScroll(), this.options.throttle));
+            window.addEventListener('scroll', helpers.throttle(() => {
+                this.setLoadOnScroll();
```

Comment:
this can be a one liner like before `window.addEventListener('scroll', helpers.throttle(() => this.setLoadOnScroll(), this.options.throttle));`